econet_hvac_furnace.yaml

```diff
@@ -46,6 +47,7 @@ econet:
             id(high_heat_lifetime_hrs).publish_state((x[132] << 8) + x[133]);
             id(blower_lifetime_hrs).publish_state((x[135] << 8) + x[136]);
             id(powered_lifetime_days).publish_state((x[137] << 8) + x[138]);
+            id(supply_air_temperature).publish_state(((x[145] << 8) + x[146])/10.0);
```

Comment:
Do you need to remove this line now that you're setting this via HWSTATUS?
```yaml
sensor_datapoint: FURNSTMP
```

Reply:
I think other folks have expressed a preference for the datapoint querying directly.  I'll comment that out and convert it to a 'template' instead of an 'econet' sensor.

Reply:
Left this datapoint in but set to request_mod:none. I believe this will let it publish updates if appropriate but not request.

Reply:
I just posted in Discord since I'm not sure I really understand this. I would be curious to hear @stockmopar take on whether this is the right approach.